### ISSUE TITLE
`xclim=0.49` breaks tests

### ISSUE THREAD

**Issue by larsbuntemeyer** [climate-service-center/index_calculator#58]
`xclim=0.49` breaks:
https://github.com/climate-service-center/index_calculator/blob/d2eac22f2d873e3d20bc446b2965bbf0e43ec14f/tests/test_indices.py#L668-L675
gives, .e.g.
```
E           AssertionError: 
E           Not equal to tolerance rtol=0.001, atol=0
E           
E           x and y nan location mismatch:
E            x: array([nan])
E            y: array(5)
```

**Comment by KatharinaBuelow** [comment]
I do not know, sorry

**Comment by larsbuntemeyer** [comment]
closes by #61
